README.md

```diff
@@ -29,3 +29,21 @@ A CloudFoundry service broker for AWS Elasticache services. Currently only Redis
 AWS_VPC_ID=vpc-deadbeef AWS_SUBNET_CIDR_BLOCK=10.0.16.0/24 \
   go run main.go -config my-config.json
 ```
+
+## Client credentials
+
+When binding a service instance to an application the *Bind* call returns the following client credentials:
+
+```
+{
+  "host": "redis-host",
+  "port": 6379,
+  "password": "pass",
+  "uri": "rediss://x:pass@redis-host:6379",
+  "tls_enabled": true
+}
+```
+
+TLS connection is always enabled and enforced.
```

Comment:
"TLS is mandatory" maybe sounds better here?

Reply:
Changed to: ```Using TLS is mandatory in the clients.```